lib/movie/screens/movie_details_screen/cubit/movie_details_cubit.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:get_it/get_it.dart';
+import 'package:movie/movie/domain/movie.dart';
+import 'package:movie/movie/domain/movie_service.dart';
+
+part 'movie_details_state.dart';
+
+class MovieDetailsCubit extends Cubit<MovieDetailsState> {
+  MovieDetailsCubit(MovieDetailsState initialState) : super(initialState);
+  MovieService get moviesService => GetIt.instance.get<MovieService>();
+
+  void getMovie(int id) async {
+    emit(MovieDetailsLoading());
+    try {
+      final movie = await moviesService.getMovieById(id);
```

Comment:
1) rename variable to "movieDetails"
2) use id from argument of MovieDetailsCubit constructor